### ISSUE TITLE
Cannot set input for zone2

### ISSUE THREAD

**Issue by Donderda** [foxriver76/ioBroker.denon#322]
**Describe the bug**
I cannot set the Input for Zone 2

**To Reproduce**
Trying to set the zone via admin in. the `objects`-Tab. Neither if the zone is powered on nor not.

**Expected behavior**
The input should be set :)

**Screenshots & Logfiles**
What exactly is needed?

**Versions:**
 - Adapter version: 1.15.2
 - JS-Controller version: 4.0.24
 - Node version: v16.19.0 
 - Operating system: Linux
 - AVR model: Denon AVR X-2100W

**Additional context**
Maybe this issue is related to this? https://github.com/foxriver76/ioBroker.denon/issues/316

**Comment by foxriver76** [comment]
Please set loglevel to debug and if you set the input in ioBroekr, it should log 

```this.log.debug(`[COMMAND] State Change - ID: ${id}; State: ${stateVal}`);```

Here the id would be interesting.

**Comment by Donderda** [comment]
Like this?

<img width="846" alt="Bildschirmfoto 2023-07-28 um 08 47 22" src="https://github.com/foxriver76/ioBroker.denon/assets/3691838/5633373f-f540-494c-b0ef-be56cb0d832f">


Setting the volume works btw.

**Comment by foxriver76** [comment]
this looks good. Then please show the `raw` object of the state. So click on the pencil behind `denon.0.zone2.selectInput` in the objects tab, then select raw and copy the whole object.

**Comment by Donderda** [comment]
Here we go :)

```
{
  "type": "state",
  "common": {
    "name": "Zone 2 Select input",
    "role": "media.input",
    "type": "string",
    "write": true,
    "read": true,
    "states": {
      "0": "PHONO",
      "1": "CD",
      "2": "TUNER",
      "3": "DVD",
      "4": "BD",
      "5": "TV",
      "6": "SAT/CBL",
      "7": "MPLAY",
      "8": "GAME",
      "9": "NET",
      "10": "SPOTIFY",
      "11": "LASTFM",
      "12": "IRADIO",
      "13": "SERVER",
      "14": "FAVORITES",
      "15": "AUX1",
      "16": "AUX2",
      "17": "AUX3",
      "18": "AUX4",
      "19": "AUX5",
      "20": "AUX6",
      "21": "AUX7",
      "22": "BT",
      "23": "USB"
    }
  },
  "native": {},
  "from": "system.adapter.denon.0",
  "user": "system.user.admin",
  "ts": 1690526607965,
  "_id": "denon.0.zone2.selectInput",
  "acl": {
    "object": 1638,
    "state": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}
```

**Comment by Donderda** [comment]
Ok, it seems that the receiver only allows to change the input to non-hdmi values...

that's odd. Do you have an idea whether it's possible to enable "all zone stereo" described here: https://manuals.denon.com/AVRX2100W/EU/EN/GFNFSYecctxnhb.php ? That would be enough for my use-case...

**Comment by Donderda** [comment]
<img width="765" alt="Screenshot 2023-07-29 at 06 45 48" src="https://github.com/foxriver76/ioBroker.denon/assets/3691838/1fe5e2a6-c145-4568-b0fb-db7e5f1b4df3">

there exists at least a telnet command for this :)

**Comment by foxriver76** [comment]
You can try to send this with `settings.expertCommand` if it does what you need we can implement it in the future as separate state

**Comment by Donderda** [comment]
Yep, it works. If I send `MNZST ON` the field `settings.surroundMode` switches to `ALL ZONE STEREO`. `MNZST OFF` sets it back again. 

Thank you for your support and your great plugin and enjoy your coffee :)

**Comment by foxriver76** [comment]
Thanks keep it open, I can implement it next time when I touch the adapter and you will get the status back too.
